connext_cmake_module/cmake/Modules/FindConnext.cmake

```diff
@@ -217,14 +216,15 @@ if(NOT "${_NDDSHOME} " STREQUAL " ")
   set(Connext_FOUND TRUE)
 else()
   # try to find_package() it
-  find_package(ndds_cpp)
-  if(ndds_cpp_FOUND)
-    message(STATUS "Found RTI Connext: ${ndds_cpp_DIR}")
-    set(Connext_INCLUDE_DIRS ${ndds_cpp_INCLUDE_DIRS})
-    set(Connext_LIBRARIES ${ndds_cpp_LIBRARIES})
+  find_package(nddscpp)
+  find_package(rticonnextmsgcpp)
+  if(nddscpp_FOUND AND rticonnextmsgcpp_FOUND)
+    message(STATUS "Found RTI Connext: ${nddscpp_DIR}")
+    set(Connext_INCLUDE_DIRS ${nddscpp_INCLUDE_DIRS})
+    set(Connext_LIBRARIES ${rticonnextmsgcpp_LIBRARIES})
```

Comment:
I agree that's weird that we find package both but use `INCLUDE_DIRS` from one and `LIBRARIES` from another. At the very least maybe @esteve could add a comment that `rticonnextmsgcpp_LIBRARIES` contains the `nddscpp_LIBRARIES` and a `TODO` comment to clean it up?